lib/tandem/tandemSimulator.js

```diff
@@ -178,6 +172,15 @@ exports.make = function(config){
     },
     suspend: function(event) {
       simpleSimulate(event);
+      setCurrStatus(event);
+    },
+    resume: function(event) {
+      ensureTimestamp(event);
+      if (currStatus !== null) {
+          event.previous = currStatus;
```

Comment:
Is there the potential for a "hall of mirrors" near-infinite regress here? Basically, I think in parallel places in other drivers with do `event.previous = _.omit(currStatus, 'previous')` to avoid having a potentially huge chain of `previous`s...